styles.css

```diff
@@ -0,0 +1,395 @@
+    animation-direction: normal;
+}
+
+.grid-item-500 {
+    display: inline-grid;
+    align-items: center;
+    border: 1px dashed #5EB1BF;
+    text-align: center;
+    font-weight: bold;
+    font-size: 30px;
+    font-family: Impact;
+    color: #f05209;
+    background-color: #042A2B;
+    animation-name: fade-in;
+    animation-duration: 2s;
+    animation-delay: 1.25s;
+    animation-iteration-count: 1;
+    animation-timing-function: linear;
+    animation-direction: normal;
+}
+
+
+.button-big {
+    display: inline-block;
+    padding: 10px;
+    padding-left: 15px;
+    padding-right: 15px;
+    background-color: #5eb1bf;
+    border: 4px solid #2c2b28;
+    border-radius: 8px;
+    font-size: 30px;
+    letter-spacing: .1em;
+    font-family: Impact;
+    color: #2c2b28;
+    box-shadow: 2px 2px #2c2b28;
+}
+
+#bounce-grow-button {
+    animation-name: bounce-grow-button;
+    animation-duration: 1s;
+    animation-delay: 2s;
+    animation-iteration-count: 6;
+    animation-timing-function: linear;
+    animation-direction: normal;
+}
+
+@keyframes bounce-grow-button {
+    0% {
+        transform: translateY(0);
+    }
+    33% {
+        transform: translateY(-5px)
+    }
+    50% {
+        padding: 15px;
+        padding-left: 20px;
+        padding-right: 20px;
+    }
+    66% {
+        transform: translateY(5px);
+    }
+    100% {
+        transform: translateY(0);
+    }
+}
+
+.button-big:hover {
+    background-color: #EF7B45;
+}
+
+.button-guess {
+    display: inline-block;
+    width: 80px;
+    padding: 5px;
+    border: 3px solid #042A2B;
+    background-color: #5eb1bf;
+    border-radius: 6px;
+    font-size: 22px;
+    letter-spacing: .1em;
+    font-family: Impact;
+    color: #042A2B;
+    text-align: center;
+}
+
+.button-guess:hover {
+    background-color: #ee7943;
+}
+
+.button-pass {
+    display: inline-block;
+    width: 80px;
+    padding: 5px;
+    border: 3px solid #5eb1bf;
+    border-radius: 6px;
+    font-size: 22px;
+    letter-spacing: .1em;
+    font-family: Impact;
+    color: #5eb1bf;
+    background-color: rgb(255, 255, 255, 0.25);
+    text-align: center;
+}
+
+.button-pass:hover {
+    background-color: white;
+}
+
+a {
+    text-decoration: none;
+    color: #042A2B;
+}
+
+.next-page-link {
+    text-align: right;
+}
+
+.next-page-link:link {
+    text-decoration: none;
+    color: #042A2B;
+}
+
+.next-page-link:visited {
+    text-decoration: none;
+    color: #5eb1bf;
+}
+
+input {
+    border: 2px solid #042A2B;
+    background-color: white;
+}
+
+#fade-in {
+    animation-name: fade-in;
+    animation-duration: 4s;
+    animation-delay: 0.25s;
+    animation-iteration-count: 1;
+    animation-timing-function: linear;
+    animation-direction: normal;
+}
+
+@keyframes fade-in {
+    0% {
+        opacity: 0;
+    }
+
+    100% {
+        opacity: 1;
+    }
+}
+
+.rounds-item:nth-of-type(3) {
+    display: inline-grid;
+    width: 1020px;
+    align-items: right;
+}
```

Comment:
Really good job, it's hard to make plain HTML/CSS look good and you did a great job!